### DISCUSSION TITLE
PSF membership

### DISCUSSION THREAD

**Discussion by hugovk** [python-pillow/Pillow#5342] [Ideas]
The Python Software Foundation is on a membership drive this month:

* https://twitter.com/ThePSF/status/1372179171082928130
* https://twitter.com/di_codes/status/1372395038815162370

Basic membership is free.

Contributing membership is for people who do at least 5 hours a month of _stuff_ to help the PSF mission ("to promote, protect, and advance the Python programming language, and to support and facilitate the growth of the international community of Python programmers"), for example, by working on open source projects (like Pillow!), and it also gives you voting rights for PSF board elections.

I joined in 2018 as a contributing member, it's really easy, you self-certify by filling in a form.

I reckon everyone in @python-pillow/pillow-team is eligible for contributing membership, as well as many of our contributors: consider joining!

Same goes for others watching this repo, as either basic or contributing!

https://www.python.org/psf/membership/#what-membership-classes-are-there

**Comment by aclark4life** [comment]
Not sure when I joined, but I am a member! Thanks @hugovk for encouraging PSF membership.

<img width="977" alt="Screen Shot 2021-03-18 at 11 32 47 AM" src="https://user-images.githubusercontent.com/72164/111652977-e38b2000-87dd-11eb-8a30-62e87a74dd0a.png">

**Reply by hugovk** [reply]
Great! Do you have basic or contributing membership?

**Reply by aclark4life** [reply]
Applied for contributing, thanks for the nudge!

<img width="684" alt="Screen Shot 2021-03-18 at 12 05 55 PM" src="https://user-images.githubusercontent.com/72164/111658059-657d4800-87e2-11eb-9a45-669c928959dd.png">
